src/services/AudiusBackend.js

```diff
@@ -452,6 +452,7 @@ class AudiusBackend {
 
       AudiusBackend.sanityChecks(audiusLibs)
     } catch (err) {
+      console.log(err)
```

Comment:
intentional?

Reply:
no harm, but should be console.error probs